drivers/eeprom/eeprom_i2c.h

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2019 Nick Brassel (tzarc)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/*
+    The base I2C address of the EEPROM.
```

Comment:
Add some documentation stating that the address needs to be shifted up by 1, as per i2c_master requirements.
The main docs state it but ppl don't read those, apparently.

Reply:
"people",  lets just say it right, drashna doesn't bother to read stuff :D

Reply:
:D